Sources/SKSupport/Path.swift

```diff
@@ -0,0 +1,22 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Basic
+import Foundation
+
+extension AbsolutePath {
+
+  /// File URL created from the normalized string representation of the path.
+  public var asURL: URL {
```

Comment:
I'd say this extension should go up to SwiftPM. I use it everywhere by myself ;-)
https://github.com/apple/swift-package-manager/pull/1894/commits

Reply:
Note: I merged this as is, but will remove our copy of this API once we update to a new enough swiftpm.